README.md

```diff
@@ -16,7 +16,7 @@ Check out [junit-pioneer.org](https://junit-pioneer.org/), particularly [the doc
 ## A Pioneer's Mission
 
 JUnit Pioneer provides extensions for [JUnit 5](https://github.com/junit-team/junit5/) and its Jupiter API.
-It does not limit itself to proven ideas with wide application but is purposely open to experiments.
+It does not limit itself to proven ideas with wide application but is purposely open to experiment.
```

Comment:
```suggestion
It does not limit itself to proven ideas with wide application but is purposely open to experiments.
```

Reply:
We should come to a a common if it should be an noun or a verb :D

Reply:
We decided in Discord to change it to "but is purposely open to experimentation"